app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AccountActivationsController < ApplicationController 
+  
+  def edit
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
+      user.update_attribute(:activated, true)
+      user.update_attribute(:activated_at, Time.zone.now)
```

Comment:
You can assign multiple attributes at the same time with:
`user.update_attributes({ activated: true, activated_at: Time.zone.now })`